tokio-postgres/src/config.rs

```diff
@@ -373,6 +375,19 @@ impl Config {
         &self.host
     }
 
+    /// Set the host used during tls. This can be useful if you are connecting to Postgres through
+    /// an ssh tunnel.
+    ///
```

Comment:
```suggestion
    /// Sets the hostname used during TLS certificate verification, if enabled.
    ///
    /// This can be useful if you are connecting through an SSH tunnel.
```